Codecademy/gamut
# style(Colors): update new blue and green shade number

### Overview
Quick follow up to this ticket: https://github.com/Codecademy/gamut/pull/2580
Context on change from 200 to 100: https://codecademy.slack.com/archives/CTVEHH2PP/p1667243744362389





### PR Checklist

- [x] Related to designs: https://www.figma.com/file/rGzb9m1sPiFihBX3u8WEmG/Catalog-revamp?node-id=828%3A46962
- [x] Related to JIRA ticket: [DISC-922]

#### Testing Instructions


Visit https://63619245a997d45475813d35--gamut-preview.netlify.app/?path=/docs/foundations-theme--swatches where you will see the new blue shade 100 and the new green shade 100.

#### PR Links and Envs

| Repository   | PR Link                                                  | PR Env                                                   |
| :----------- | :------------------------------------------------------- | :------------------------------------------------------- |
| Monolith     | [Monolith PR](https://github.com/codecademy-engineering/Codecademy/pull/33327)  | [Monolith Env](https://pr-33327-monolith.dev-eks.codecademy.com/) |
| Monorepo | [Another Link](https://github.com/codecademy-engineering/mono/pull/523) | [Another Env]( https://tengu.codecademy.com/catalog?PR_ENV=portal-app-pr-523)  |




[DISC-922]: https://codecademy.atlassian.net/browse/DISC-922?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
